lms/static/js/edxnotes/plugins/accessibility.js

```diff
@@ -0,0 +1,253 @@
+;(function (define, undefined) {
+'use strict';
+define(['jquery', 'underscore', 'annotator_1.2.9'], function ($, _, Annotator) {
+    /**
+     * Adds the Accessibility Plugin
+     **/
+    Annotator.Plugin.Accessibility = function () {
+        // Call the Annotator.Plugin constructor this sets up the element and
+        // options properties.
+        Annotator.Plugin.apply(this, arguments);
+    };
+
+    $.extend(Annotator.Plugin.Accessibility.prototype, new Annotator.Plugin(), {
+        pluginInit: function () {
+            this.annotator.subscribe('annotationViewerTextField', _.bind(this.addAriaAttributes, this));
+            this.annotator.element.on('keydown', '.annotator-hl', _.bind(this.onHighlightKeyDown, this));
+            this.annotator.element.on('keydown', '.annotator-viewer', _.bind(this.onViewerKeyDown, this));
+            this.annotator.element.on('keydown', '.annotator-editor', _.bind(this.onEditorKeyDown, this));
+            this.addTabIndex();
+        },
+
+        destroy: function () {
+            this.annotator.unsubscribe('annotationViewerTextField', this.addAriaAttributes);
+            this.annotator.element.off('keydown', '.annotator-hl');
+            this.annotator.element.off('keydown', '.annotator-viewer');
+            this.annotator.element.off('keydown', '.annotator-editor');
+        },
+
+        addTabIndex: function () {
+            var controls, edit, del;
+            controls = this.annotator.element.find('.annotator-controls');
+            edit = controls.find('.annotator-edit');
+            edit.attr('tabindex', 0);
+            del = controls.find('.annotator-delete');
+            del.attr('tabindex', 0);
+        },
+
+        addAriaAttributes: function (field, annotation) {
+            var ariaNoteId = 'aria-note-' + annotation.id;
+            // Add ARIA attributes to highlighted text ie <span class="annotator-hl">Highlighted text</span>
+            // aria-describedby refers to the actual note that was taken.
+            _.each(annotation.highlights, function(highlight) {
+                $(highlight).attr('aria-describedby', ariaNoteId);
```

Comment:
@polesye 
> aria-describedby doesn't work, because corresponding element is created dynamically when the dialog opens.

Yes, it unfortunately doesn't work. @cptvitamin Any suggestion here?